### PR TITLE
🐛 fix 404 routing for nested routes

### LINKED ISSUE #637: 🐛 404 not working for nested routes
Going to `/cb/does-not-exist` or `visitor/whereami` does not result in a 404 due to clientside routing changes

### PR DESCRIPTION
**Pull request by astroash**
Fixes #637 

#### Changes
* nested routes use switch statement so they can return 404 if no matching page is found

#### User testing flows:

- [x] `/cb/does-not-exist` returns 404 page
- [x] `/visitor/does-not-exist` returns 404 page

### CODE REVIEW

File: src/visitors/index.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-  <Fragment>
+  <Switch>
     <Route exact path="/visitor/home" component={Home} />
     <Route path="/visitor/signup" component={Signup} />
     <Route exact path="/visitor/login" component={Login} />
     <Route exact path="/visitor/qrerror" component={QrError} />
     <Route exact path="/visitor/end" component={redirectAfterTimeout('/visitor/home', 5000)(Thanks)} />
     <Route exact path="/visitor/thankyou" component={redirectAfterTimeout('/visitor/home', 5000)(ThanksFeedback)} />
-  </Fragment>
+    <Route path="/visitor" component={NotFound} />
```
Comment (eliascodes) [inline src/visitors/index.js]: Here we specify `path`, but in the CB routes we don't. What's the difference, and which should we prefer?

Comment (astroash) [reply src/visitors/index.js]: Good spot, the path is already specified by the parent router so this should be removed.

Comment (astroash) [reply src/visitors/index.js]: fixed in amend

**Comment by eliascodes** [comment]
Should I squash this one instead?

**Comment by astroash** [comment]
You can't squash one commit.

**Pull request merged**